news/get-authors.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
```

Comment:
Do we need to run this before towncrier to generate the authors for a release? If so it'd be good to note that in the release documentation. It'd be cool to auto-generate the contributors to issues and PRs by looking at the news items and querying the PRs/issues for users as well, since bug reports are valuable contributions.

Reply:
My original intention was that new `.author` files would be added as part of the PR (just like the news file). But I left this script in case we forgot people and wanted to check before the release.
I agree it'd be cool to get contributors from the GitHub API too, I'll see if I can add it in the future.

Reply:
Cool, sounds good to me :smile: